consultation_analyser/consultations/models.py

```diff
@@ -1,7 +1,51 @@
-# Create your models here.
+import uuid
 from django.db import models
 
 
-class Question(models.Model):
+class UUIDPrimaryKeyBase(models.Model):
+    class Meta:
+        abstract = True
+
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+
+
+class SlugPrimaryKeyBase(models.Model):
+    class Meta:
+        abstract = True
+
+    slug = models.CharField(primary_key=True, blank=False, max_length=256)
+
+
+class Consultation(SlugPrimaryKeyBase):
+    name = models.CharField(max_length=256, blank=False)
+
+
+class Section(SlugPrimaryKeyBase):
+    consultation = models.ForeignKey(Consultation, on_delete=models.CASCADE, null=True)
+    text = models.TextField(blank=True)
+
+
+class Question(SlugPrimaryKeyBase):
     text = models.CharField(max_length=None)  # no idea what's a sensible value for max_length
-    slug = models.CharField(max_length=256)
+    has_free_text = models.BooleanField(default=False)
+    multiple_choice_options = models.JSONField(default=list)
+    consultation = models.ForeignKey(Consultation, on_delete=models.CASCADE, null=True)
+
+
+class Respondent(UUIDPrimaryKeyBase):
```

Comment:
Final comment, sorry!!

I think we should have a `response` rather than a `respondent`. (We can have a `respondent` too in due course)

Reply:
Name has now been changed to ConsultationResponse.